pkg/openstack/loadbalancer.go

```diff
@@ -74,6 +74,7 @@ const (
 	ServiceAnnotationLoadBalancerFloatingSubnetTags   = "loadbalancer.openstack.org/floating-subnet-tags"
 	ServiceAnnotationLoadBalancerClass                = "loadbalancer.openstack.org/class"
 	ServiceAnnotationLoadBalancerKeepFloatingIP       = "loadbalancer.openstack.org/keep-floatingip"
+	ServiceAnnotationLoadBalancerFloatingIP           = "loadbalancer.openstack.org/floating-ip"
```

Comment:
in order to comply with the previous naming, I'd suggest to use `loadbalancer.openstack.org/floatingip`
in future for dual stack we can use `loadbalancer.openstack.org/floatingip-v6`